keyboards/keebzdotnet/wazowski/keymaps/via/keymap.c

```diff
@@ -0,0 +1,30 @@
+/* Copyright 2020 keebzdotnet
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+// Defines names for use in layer keycodes and the keymap
+enum layer_names {
+    _BASE
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    /* Base */
+    [_BASE] = LAYOUT(
+        KC_A, KC_B, KC_C,KC_D, KC_E,
+        KC_F, KC_G, KC_H, KC_I, KC_J,
+        KC_K, KC_L, KC_NO, KC_M, KC_N
```

Comment:
```suggestion
        KC_K, KC_L, KC_NO, KC_M
```

Reply:
@zvecr hello! I was able to compile and test this code and it had been working. I think that maybe I was using an outdated version of QMK or something. I've pushed a commit to fix it. I've tested it on my built Wazowski 23-19 and its working. This is my first time writing code for QMK and VIA lol